Gemfile.base

```diff
@@ -10,12 +10,14 @@ gem 'aws-sdk', '~> 2'
 gem 'aws-sdk-rails', '~> 1.0'
 
 gem 'dotenv-rails', '~> 2.7'
-gem 'rails', '4.2.11.1'
-#gem 'rails', '5.0.6'
-#gem 'activesupport', '4.2.9' # somehow paperclip needs to explicitly lock the version to not use 5.0
+#gem 'rails', '4.2.9'
```

Comment:
Let's be optimistic 😉

```suggestion
```

Reply:
I agree with Gui, besides it feels strange that it is not even the version that we have been using (`4.2.11.1`) but a previous one.